civitai-downloader-v2/src/cli/main.py

```
@@ -109,68 +110,783 @@ def cli(ctx, config, verbose):
+                        if show_versions and base_model:
+                            # Version display mode - output each matching version
+                            model_id = result.get("id", 0)
+                            model_name = result.get("name", "Unknown")
+                            model_type = result.get("type", "Unknown")
+                            model_versions = result.get("modelVersions", [])
+                            
+                            # Filter and show only matching versions
+                            for version in model_versions:
+                                if isinstance(version, dict):
+                                    version_base = version.get("baseModel", "")
+                                    if version_base.lower() == version_base_model.lower():
```

Ruff (0.12.2)

Undefined name `version_base_model`

(F821)